impl.go

```diff
@@ -305,6 +320,32 @@ func validReceiver(recv string) bool {
 	return err == nil
 }
 
+// commentsBefore returns true if comments precede the field. It returns false
```

Comment:
`// commentsBefore reports whether commentGroups precedes field.`

Maybe s/commentGroups/cg/ throughout this doc and func.

Reply:
Done

Reply:
Doesn't look quite done. (Maybe github diff is misleading me, though?) Please use the "reports whether" formulation:

`// commentsBefore reports whether cg precedes field.`

Reply:
I think this might have gotten lost in the fray. Mind updating the wording here?